### ISSUE TITLE
Training Knowledge distillation with model trained on Fairseq

### ISSUE THREAD

**Issue by Raghava14** [pytorch/translate#401]
Hi ,
I have trained a En-De model using fairseq and now want to try the knowledge_distillation task to train the student model on [reduced transformer architecture](https://github.com/pytorch/fairseq/blob/master/fairseq/models/transformer.py#L875).I was however unable to do so. I wanted to know whether this is possible i.e., training teacher on fairseq and student in translate libraries. Or should the teacher be also trained on translate library. Kindly clarify. Also I am trying to do knowledge_distillation task and I am confused on how to give inputs.
Say I have model , binaries and dictionaries generated from walkthrough in fairseq. How can I use them from knowledge distillation. Are they enough or should I also need the logits saved in file and how should I use them please elaborate.
Thanks

**Comment by rasoolims** [comment]
Hi,

Sorry for the late reply.

FairSeq is not compatible with Pytorch/translate models. Our KD code is still experimental and there is not guarantee that it works as expected.

Thanks
